validation/src/main/scala/hmda/validation/rules/lar/validity/V570.scala

```diff
@@ -0,0 +1,21 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.{ Failure, Result }
+import hmda.validation.rules.EditCheck
+
+object V570 extends EditCheck[LoanApplicationRegister] {
+  override def name: String = "V570"
+
+  override def apply(lar: LoanApplicationRegister): Result = {
+    when(lar.lienStatus is equalTo(1)) {
+      lar.rateSpread is equalTo("NA") or {
+        try {
```

Comment:
Good question. If I remember correctly, we eventually need to be sending the official HMDA-sanctioned error message for each edit, and we aren't currently set up to do that. So, my understanding is that the current error messages will be removed via refactor in the future (but not sure when).